PostmanTests/UserService.postman_collection.json

```diff
@@ -50,6 +50,144 @@
 				}
 			],
 			"protocolProfileBehavior": {}
+		},
+		{
+			"name": "CreateUser",
+			"item": [
+				{
+					"name": "With correct body",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "6e99293a-94f5-4726-9781-a4a80260899b",
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"pm.test(\"Body matches string\", function () {",
+									"    pm.expect(pm.response.text()).to.include(\"true\");",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n\t\"Email\" : \"email2@yandex.ru\",\n\t\"Password\" : \"123\",\n\t\"FirstName\" : \"Lep\",\n\t\"LastName\" : \"Kor\",\n\t\"Birthday\" : \"2020-04-05\"\n}",
```

Comment:
difficult to read

Reply:
it is how postman format collection, no need for any changes here